tests/test_iib_ops.py

```diff
@@ -357,6 +357,48 @@ def test_add_bundles_py(
     fixture_pulplib_repo_publish.assert_called_once()
 
 
+def test_add_bundles_py_multiple_bundles(
+    fixture_iib_client,
+    fixture_pulp_client,
+    fixture_iib_krb_auth,
+    fixture_pulplib_repo_publish,
+    fixture_pulplib_repo_sync,
+    fixture_container_image_repo,
+    fixture_common_iib_op_args,
+):
+
+    repo = fixture_container_image_repo
+    fixture_pulp_client.return_value.search_repository.return_value = [repo]
+    fixture_pulp_client.return_value.get_repository.return_value = repo
+    with setup_entry_point_py(
+        ("pubtools_iib", "console_scripts", "pubtools-iib-add-bundles"),
+        {"PULP_PASSWORD": "pulp-password", "CNR_TOKEN": "cnr_token"},
+    ) as entry_func:
+        retval = entry_func(
+            ["cmd"]
+            + fixture_common_iib_op_args
+            + ["--bundle", "bundle1", "--bundle", "bundle2"]
```

Comment:
Although it seems to accept both values, the JIRA description says the command "should be `--bundle bundle1 bundle2`", as the help text for the arg describes it as a "comma separated list".

Reply:
There is an inconsistency in how pub constructs pubtools-iib arguments vs how pubtools-iib accepts them. Pub constructs them as '--bundle bundle1 --bundle bundle2' while pubtools-iib accepts them as '--bundle bundle1 bundle2'. This causes a bug where passing '--bundle bundle1 --bundle bundle2' makes pubtools-iib accept only the second one.

My original solution was to modify pub to create the arguments in a way that pubtools-iib accepts them, but Jindrich said that I should actually change the way pubtools-iib accepts them, so here we are.

The comma separated values refers to how pub accepts this argument, so it doesn't have anything to do with this PR.

Reply:
Ah, okay. The JIRA refers to the iib command, so maybe that should be updated.

Reply:
+1 to updating JIRA